src/index.html

```diff
@@ -1,13 +1,330 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
+    <title>MyBike</title>
+    <link
+    rel="shortcut icon"
+    href="images/favicon.png"
+    type="image/x-icon"
```

Comment:
```suggestion
      rel="shortcut icon"
      href="images/favicon.png"
      type="image/x-icon"
```
Consider fixing indents according to the code style guide